pkg/responses.go

```diff
@@ -65,6 +65,10 @@ const (
 		DeletePreviewSite + " - deletes the preview site\n" +
 		"```"
 
+	// Preview denied is a message that preview bot sends when user doesn't have necessary capability
+	// logs will contain more information about the reason why user has been denied if it is anything else except user missing SiteEditor capability
+	previewDenied = "**Unauthorized** preview site command, please contact your DDEV-Live organization administrator for `%v`"
```

Comment:
I know this is a hotly debated topic, but are we continuing to use "organization" until further notice?

Reply:
definitely open to suggestions here, not sure what wording would make the most sense. Also tagging @alkymst @cyberswat @jdarrohn @drekle @josefkarasek @ekl1773 @dclear @slivermon, appreciate everyone's input here.

Reply:
I think we can sidestep that verbiage problem by saying something like

`please contact your DDEV-Live account owner for `%v`"`

Reply:
thanks guys, I like that wording.